### PR TITLE
add windows client assistance request control menu item

### PR DESCRIPTION
**Pull request by garbb**
image

### CODE REVIEW

File: client/Windows/wf_client.c [hunk 1]
```diff
@@ -272,6 +272,29 @@ static BOOL wf_pre_connect(freerdp* instance)
 	return TRUE;
 }
 
+static void wf_append_item_to_system_menu(HMENU hMenu, UINT fMask, UINT wID, const wchar_t* text,
+                                          void* dwItemData)
+{
+	// initial windows system item position where we will insert new menu item
```
Comment (hardening) [inline client/Windows/wf_client.c:278]: Change the comment to C comments, not C++

File: client/Windows/wf_client.c [hunk 2]
```diff
@@ -272,6 +272,29 @@ static BOOL wf_pre_connect(freerdp* instance)
...
+	// after default 5 items (restore, move, size, minimize, maximize)
+	// gets incremented each time this function is called
+	// or maybe could use GetMenuItemCount() to get initial item count ?
+	static UINT position = 6;
```
Comment (hardening) [inline client/Windows/wf_client.c:282]: not a big fan of hidden global variables, can't you store that somewhere in wfContext ?

Comment (garbb) [reply client/Windows/wf_client.c]: ok

File: client/Windows/wf_client.c (hunk 1 above)
Comment (garbb) [reply client/Windows/wf_client.c]: Sorry I do not understand what this means. Do you mean comments `/* like this */` ? I changed the comment to this format.

Comment (hardening) [reply client/Windows/wf_client.c]: Yes unfortunately some (old) compiler don't like C++ style comment in plain C

**Comment by hardening** [comment]
@freerdp-bot test please

**Review by hardening — APPROVED** [review]
LGTM (but I'm not the perfect expert of the subject)

File: client/Windows/wf_client.c [hunk 3]
```diff
@@ -272,10 +272,26 @@ static BOOL wf_pre_connect(freerdp* instance)
 	return TRUE;
 }
 
+static void wf_append_item_to_system_menu(HMENU hMenu, UINT fMask, UINT wID, const wchar_t* text,
+                                          wfContext* wfc)
+{
+	MENUITEMINFO item_info;
+
+	ZeroMemory(&item_info, sizeof(MENUITEMINFO));
```
Comment (akallabeth) [inline client/Windows/wf_client.c:280]: better use `MENUITEMINFO item_info = { 0 };`

**Comment by akallabeth** [comment]
@freerdp-bot test

**Comment by akallabeth** [comment]
@garbb could you rebase the branch and squash all commits into one please?

**Comment by garbb** [comment]
well i think i finally figure out how to merge all the commits after i messed it up several times??

**Comment by akallabeth** [comment]
@freerdp-bot test

**Comment by akallabeth** [comment]
@freerdp-bot please test again!

**Comment by akallabeth** [comment]
@freerdp-bot test please

**Comment by akallabeth** [comment]
@freerdp-bot test

**Pull request merged**